activities/codebase-stewardship/supporting-governance/governance-template.md

```diff
@@ -0,0 +1,87 @@
+---
+type: Resource
+---
+
+# Template for a simple governance file
+
```

Comment:
This template is missing context, which means it's not self-explanatory or usable if someone finds it on its own. Here's an example of a template where we have provided context: 
https://github.com/publiccodenet/about/blob/develop/activities/codebase-stewardship/assessment-issue-template.md

Reply:
@Ainali - can you add context?

Reply:
Perhaps we should mark this for improvement and then continue, as this is better to solve in the develop than in the release branch probably.

Reply:
Resolved in #599